src/views/public/analytics/Home.vue

```diff
@@ -1,32 +1,32 @@
 <template>
-<div class="lg:pr-2 xl:pr-0 xl:flex pb-32">
+<div class="px-4 lg:pl-0 lg:pr-2 xl:pr-0 xl:flex pb-32">
   <div class="w-full xl:w-2/3">
-    <header class="flex justify-between">
-      <h5 class="text-4xl">
+    <header class="flex flex-wrap lg:flex-no-wrap justify-between">
+      <h5 class="w-full text-4xl mb-6 lg:mb-0">
         Analytics
       </h5>
-      <div class="flex justify-between items-center">
+      <div class="w-full flex items-center">
         <button class="border-black border py-1 w-32 text-center font-circular mr-3">Traffic</button>
         <button class="border-black border py-1 w-32 text-center font-circular">Signups</button>
       </div>
     </header>
-     <div class="stats flex mt-12">
-      <div class="stat flex-grow">
+     <div class="stats flex flex-wrap mt-6 lg:mt-12">
+      <div class="stat w-1/2 lg:w-1/4">
         <p class="stat-title font-semibold font-circular">Current Leaders</p>
         <p class="stat-data">6</p>
         <!-- <p class="stat-data">{{statistics.currentLeaders}}</p> -->
       </div>
-      <div class="stat flex-grow">
+      <div class="stat w-1/2 lg:w-1/4 mb-4 lg:mb-0">
         <p class="stat-title font-semibold font-circular">Upcoming Leaders</p>
         <p class="stat-data">6</p>
         <!-- <p class="stat-data">{{statistics.upcomingLeaders}}</p> -->
       </div>
-      <div class="stat flex-grow">
+      <div class="stat w-1/2 lg:w-1/4">
         <p class="stat-title font-semibold font-circular">Parties</p>
         <p class="stat-data">6</p>
         <!-- <p class="stat-data">{{statistics.parties}}</p> -->
       </div>
-      <div class="stat flex-grow pl-10 border-l-2 border-gray-c4">
+      <div class="stat w-1/2 lg:w-1/4 lg:pl-10 lg:border-l-2 border-gray-c4">
         <p class="stat-title font-semibold font-circular">Admins</p>
         <p class="stat-data">6</p>
         <!-- <p class="stat-data">{{statistics.admins}}</p> -->
```

Comment:
are we sticking to static data and not live?

Reply:
@bolorundurowb I'll leave this for @adetola-ralph

Reply:
We're going to use static data where we do not have actual data. I will update as we get live data @bolorundurowb @Saint-lawal